src/renderer/mod.rs

```diff
@@ -771,7 +771,16 @@ impl<'a> RenderApi<'a> {
                     fg: Color::Spec(Rgb { r: 0, g: 0, b: 0}),
                     flags: cell::Flags::empty(),
                 }
-            })
+            }, IndexedCell {
```

Comment:
With this I'm not exactly sure whether it's right to do or not, because since I use this `type Item = (IndexedCell, IndexedCell);` I had no other option and for `render_cells` it's `where I: Iterator<Item=(::term::IndexedCell, ::term::IndexedCell)>`.